### ISSUE TITLE
Differentiation section x_squared function

### ISSUE THREAD

**Issue by PyLief** [NU-CEM/Computational_Physics#11]
In "Finite difference methods" differentiation section, the function called "x_squared" is actually "2 * x_squared." Either change function name or correct output throughout page please.

**Comment by lucydot** [comment]
Yes, this needs fixing! I'll update now. Thanks for highlighting @PyLief  👍

**Comment by lucydot** [comment]
@PyLief the website has been updated. If you agree this fixes the problem, let me know and I'll close the issue.
